source/Halibut.Tests/TraceLogFileLogger.cs

```diff
@@ -55,7 +55,7 @@ async Task WriteDataToFile()
                 // So what we can write it down as one chunk.
                 while (queue.TryDequeue(out var log)) list.Add(log);
 
-                using(var fileWriter = new FileStream(logFilePath, FileMode.Append, FileAccess.Write, FileShare.Delete | FileShare.ReadWrite))
+                using(var fileWriter = await OpenLogFile())
```

Comment:
Just wondering, was this an issue locally or on TeamCity?

Reply:
locally, my guess is crowd strike with slightly different timings (I had turbo on my laptop turned off, so I guess that increased the time files were opened for which I guess resulted in the conflict)